src/jquery.payment.coffee

```diff
@@ -150,12 +150,29 @@ safeVal = (value, $target) ->
     $target.prop('selectionStart', cursor)
     $target.prop('selectionEnd', cursor)
 
+# Replace Full-Width Chars
+
+replaceFullWidthChars = (str = '') ->
+  fullWidth = '\uff10\uff11\uff12\uff13\uff14\uff15\uff16\uff17\uff18\uff19'
```

Comment:
Yup! The last half of the second byte corresponds to the numeral, if you want a quick way of identifying it. `'\uff14' == '４'`